dbcon/mysql/ha_mcs_client_udfs.cpp

```diff
@@ -924,6 +924,103 @@ extern "C"
   {
   }
 
+  // --- Plan UDFs ---
+  // Return plan strings or applied rules based on optional parameter:
+  //   no arg or 1 -> optimized/latest plan (ci->queryPlan)
+  //   0 or 'original'  -> pre-RBO plan
```

Comment:
optimizer, I would say b/c when we add CBO nobody will change this line.

Reply:
I didn't parse your comment

Reply:
```suggestion
  //   0 or 'original'  -> plan before optimizer
```